resources/assets/js/store/Job/jobSelectorComplex.ts

```diff
@@ -0,0 +1,209 @@
+import createCachedSelector from "re-reselect";
+import { uniq, difference, intersection } from "lodash";
```

Comment:
Same thing here, you can import them all directly, although they have to be on individual lines.

Reply:
08ab54cb35be20b3de917d97c1bf859c40799db4